tests/testthat/test-validate-robin_glm.R

```diff
@@ -0,0 +1,55 @@
+library(RobinCar)
+
+# Compare RobinCar2 to RobinCar results
+# -------------------------------------
+
```

Comment:
Check whether we should do skip_on_CRAN (or something) to avoid having to install RobinCar when this testing suite is run.

Reply:
no need to skip at the moment